src/RoslynInsertionTool/scripts/VS-Insertion.ps1

```diff
@@ -59,10 +59,12 @@ Do-Insertion -component "Roslyn"            -queueName "Roslyn-Signed"         -
 Do-Insertion -component "VS Unit Testing"   -queueName "VSUnitTesting-Signed"  -fromBranch "dev16.0"                  -toBranch "rel/d16.0"    -insertCore "true"     -queueValidation "true" -dropPath "server"
 
 # Dev16.1 Preview 2
-Do-Insertion -component "Roslyn"            -queueName "Roslyn-Signed"         -fromBranch "dev16.1-preview2-vs-deps" -toBranch "rel/d16.1"    -insertToolset "true"  -insertDevdiv "false"   -updatecorextlibraries "true" -queueValidation "true"
+#Do-Insertion -component "Roslyn"            -queueName "Roslyn-Signed"         -fromBranch "dev16.1-preview2-vs-deps" -toBranch "rel/d16.1"    -insertToolset "true"  -insertDevdiv "false"   -updatecorextlibraries "true" -queueValidation "true"
 #Do-Insertion -component "Live Unit Testing" -queueName "TestImpact-Signed"     -fromBranch "master"                   -toBranch "lab/d16.1stg" -insertCore "false"    -insertDevdiv "false"   -queueValidation "true"
 #Do-Insertion -component "VS Unit Testing"   -queueName "VSUnitTesting-Signed"  -fromBranch "dev16.1.x"                -toBranch "lab/d16.1stg" -insertCore "true"     -queueValidation "true" -dropPath "server"
 
 # Dev16.1 Preview 3
-Do-Insertion -component "Roslyn"            -queueName "Roslyn-Signed"         -fromBranch "master-vs-deps"           -toBranch "lab/d16.1stg" -insertToolset "true"  -insertDevdiv "false"   -updatecorextlibraries "true" -queueValidation "true"
 Do-Insertion -component "VS Unit Testing"   -queueName "VSUnitTesting-Signed"  -fromBranch "master"                   -toBranch "lab/d16.1stg" -insertCore "true"     -queueValidation "true" -dropPath "server"
+Do-Insertion -component "Roslyn"            -queueName "Roslyn-Signed"         -fromBranch "dev16.1-preview3-vs-deps" -toBranch "lab/d16.1stg" -insertToolset "true"  -insertDevdiv "false"   -updatecorextlibraries "true" -queueValidation "true"
```

Comment:
Into `lab/d16.1stg` just until the divisional snap, when it'll change to `rel/d16.1`.